t/certs/miss_head.crt

```diff
@@ -0,0 +1,25 @@
+MIIEojCCAwqgAwIBAgIJAK253pMhgCkxMA0GCSqGSIb3DQEBCwUAMFYxCzAJBgNV
```

Comment:
You don't have to introduce yet another bad cert and key just for mimicking bad base64 decoded strings. Do it by literal strings.

Reply:
OK.